Project-Timer-frontend/src/app/slider/slider.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-slider',
+  templateUrl: './slider.component.html',
+  styleUrls: ['./slider.component.scss']
+})
+export class SliderComponent implements OnInit {
+
+  constructor() { }
```

Comment:
jeśli coś jest puste lub niepotrzebne w tym momencie to wywal

Reply:
done